lerna.json

```diff
@@ -1,5 +1,5 @@
 {
   "$schema": "node_modules/lerna/schemas/lerna-schema.json",
   "useWorkspaces": true,
-  "version": "0.2.0"
+  "version": "independent"
```

Comment:
From the doc, I read 

> Independent mode Lerna projects allows maintainers to increment package versions independently of each other. Each time you publish, you will get a prompt for each package that has changed to specify if it's a patch, minor, major or custom change.

How will this "prompt" be handled in the github action ?

Reply:
It is assumed we let lerna take control over the versioning. In the future we might add a customized versioning manager. To be discussed.

Reply:
using conventional commits to handle versioning automatically require to strictly follow the convention. few devs are aware of it, especially the BREAKING_CHANGE rule. I'm not sure we want to automatically release a new version on each merge